packages/ingest/granule.js

```diff
@@ -184,6 +184,7 @@ class Granule {
 
     return {
       granuleId: granule.granuleId,
+      dataType: granule.dataType,
```

Comment:
Why is this being modified?

Reply:
To pass the dataType down stream.  It is needed by another instance of queue-granules to pass the granule to the processing workflow